helm/kanister-operator/templates/deployment.yaml

```diff
@@ -41,6 +41,8 @@ spec:
           value: {{ .Values.controller.logLevel }}
         - name: DATA_STORE_PARALLEL_UPLOAD
           value: {{ .Values.controller.parallelism | quote }}
+        - name: DATA_STORE_PARALLEL_DOWNLOAD
+          value: {{ .Values.controller.parallelism | quote }}
```

Comment:
We are referring to the same flag `controller.parallelism` for upload and download parallelism. Is it expected? Or should we have separate flags for both?

Reply:
I've intentionally took it from the same value, cause this is very specific settings. But since you both think that it's good to have them separated, not a big deal. I can introduce new value.

The question is, how new value should be named ? I'd go for `.Values.controller.parallelDownloads` but in such case the next question will follow up - what to do with old one. Should it be renamed to `parallelUploads` ? If yes - should we notify users about that change (parallelism is not mentioned in documentation, and as far as I understand it's undocumented feature) ?

Reply:
I was thinking if it makes sense to have separate section for datastore related config? Having it at controller level seems bit confusing. Controller also perform other operations like actionset action executions. 
Maybe something like 
```
dataStore:
  parallelism:
    upload:
    download:
```

Reply:
> If yes - should we notify users about that change (parallelism is not mentioned in documentation, and as far as I understand it's undocumented feature)

This feature is not usable right now. It doesn't work with Kanister without Kopia server setup. Which we haven't documented yet. So we have the privilege to change it without announcing it.

Reply:
Thank you ! Great idea, done.